src/encoded/static/components/static-pages/components/StatisticsPageViewBody.js

```diff
@@ -0,0 +1,1092 @@
+
+                    <hr className="mt-3"/>
+                    {/*
+                    <div className="mb-15">
+                        <div className="text-400 inline-block">Grouping by&nbsp;&nbsp;</div>
+                        <DropdownButton id="select_fields_faceted_group_by" onSelect={this.changeFieldFacetedByGrouping}
+                            title={<span className="text-500">{ UsageStatsView.fieldsFacetedByOptions[fields_faceted_group_by] }</span>}>
+                            { _.map(_.pairs(UsageStatsView.fieldsFacetedByOptions), ([ key, title ]) =>
+                                <DropdownItem eventKey={key} key={key}>{ title }</DropdownItem>
+                            ) }
+                        </DropdownButton>
+                    </div>
+                    */}
+
+                    <AreaChartContainer {...commonContainerProps} id="fields_faceted"
+                        title={
+                            <h4 className="text-300 mt-0">
+                                <span className="text-500">Top Fields Faceted</span> { countBy.fields_faceted === 'sessions' ? '- by user session' : '- by search result instance' }
+                            </h4>
+                        }
+                        extraButtons={<UsageChartsCountByDropdown {...countByDropdownProps} chartID="fields_faceted" />}>
+                        <AreaChart {...commonChartProps} data={fields_faceted} />
+                    </AreaChartContainer>
+
+                    <HorizontalD3ScaleLegend {...{ loadingStatus }} />
+
+                </ColorScaleProvider>
+
+                : null }
+
+        </div>
+    );
+});
+
+
+export class SubmissionsStatsView extends React.PureComponent {
+
+    /**
+     * Use this only for charts with child terms 'Internal Release' and 'Public Release', which are
+     * meant to have a separate color scale and child terms from other charts.
+     *
+     * @param {string} term - One of 'Internal Release' or 'Public Release'.
+     * @returns {string} A CSS-valid color string.
+     */
+    static colorScaleForPublicVsInternal(term){
+        if (term === 'Internal Release' || term === 'Internally Released'){
+            return '#ff7f0e'; // Orange
+        } else if (term === 'Public Release' || term === 'Publicly Released'){
+            return '#1f77b4'; // Blue
+        } else {
+            throw new Error("Term supplied is not one of 'Internal Release' or 'Public Release': '" + term + "'.");
+        }
+    }
+
+    render(){
+        const {
+            loadingStatus, mounted, session, currentGroupBy, groupByOptions, handleGroupByChange, windowWidth,
+            // Passed in from StatsChartViewAggregator:
+            expsets_released, expsets_released_internal, files_released, file_volume_released,
+            expsets_released_vs_internal, /* expsets_created, */ chartToggles, smoothEdges, width, onChartToggle, onSmoothEdgeToggle
```

Comment:
Personally, I really don't like inline commented out code like this unless there is a really good reason for it (or JS standard thinks its okay, which it might).